internal/http/handlers.go

```diff
@@ -49,6 +51,36 @@ func MetaHandler(cfg *config.Config) http.HandlerFunc {
 	}
 }
 
+func ListHandler(config *config.Config, database db.DB, assets Assets) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		var err error
+
+		log := logger.From(r.Context())
+
+		buff := bytes.NewBuffer([]byte{})
+		assets.Template().ExecuteTemplate(buff, "list.go.html", nil)
+		html, _ := io.ReadAll(buff)
+
+		vars := map[string]interface{}{
+			"FileID":    "Latest snips",
+			"FileSize":  nil,
+			"CreatedAt": nil,
+			"UpdatedAt": nil,
+			"FileType":  nil,
+			"RawHREF":   "",
+			"HTML":      template.HTML(string(html)),
```

Comment:
Instead of sending mostly empty html, I'd prefer if we had this list server side rendered.